packages/block-library/src/list-item/edit.js

```diff
@@ -40,6 +45,7 @@ function IndentUI( { clientId } ) {
 				icon={ isRTL() ? formatIndentRTL : formatIndent }
 				title={ __( 'Indent' ) }
 				describedBy={ __( 'Indent list item' ) }
+				shortcut={ _x( 'Space', 'keyboard key' ) }
```

Comment:
I don't think we should add this. Pressing space is indenting only if we are at the start of the list item and might be confusing, no?

Reply:
Hi @ntsekouras, Currently the list block does show the space shortcut and it only works if we are at the start. I'm keeping the same behavior in this PR.

Reply:
I saw that it was in the old version, but still don't think it makes much sense. Do you have strong opinion on keeping it? If yes, I'd like to know the reason. This is not going to be a blocker for me of course, but would like to hear your thoughts.

Reply:
> Do you have strong opinion on keeping it?

No, I was just following what we had, I removed the shortcut because I agree it is confusing.